packages/runtime/container-runtime/src/connectionTelemetry.ts

```diff
@@ -302,6 +325,18 @@ class OpPerfTelemetry {
 				opPerfData: {},
 			});
 		}
+
+		if (message.type == MessageType.NoOp) {
+			// Count the number of no-ops submitted by this client.
+			// The value is reset when we log the OpStats sampled event.
+			this.noOpCount++;
+		}
+
+		this.opsLogger.sendPerformanceEvent({
```

Comment:
When looking at the telemetry, we divide `submitOpCount` by `noOpCount` to get how many ops per noop right? Maybe we can add that ratio as a property for easier consumption?